maintain/scale-up.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Adding storage to existing PX Cluster Nodes"
 keywords: scale-up
 sidebar: home_sidebar
-redirect_from: "/scale-up.html"
+redirect_from:
+  - /scale-up.html
+  - /convert-x86-to-storage-nodes.html
```

Comment:
I thought it was an old page that is no longer used, I tried to find it's another similar one